.github/workflows/maven-build.yml

```diff
@@ -19,9 +19,9 @@ jobs:
     name: build-only (Java ${{ matrix.java }})
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: false
+      fail-fast: true
```

Comment:
I don't think you should change that. It doesn't change anything if you have only one element and if we introduce more, we will want `false`.